airbot_sensor_to_pointcloud:
  ros__parameters:
    target_frame: "map" # "map" or "base_link"

    use:
      tof: True
      tof_1D: True
      tof_left: True
      tof_right: True
      tof_row: True
      camera: True
      cliff: True

    camera:
      pointcloud_resolution: 0.05
      class_id_confidence_th: # 형식: "class id: confidence score"
        - "2: 55"
        - "5: 55"
        - "6: 55"
      object_direction: False  # 정방향(CCW+):True, 역방향(CW+):False

    publish:
      rate_ms:
        tof_1d: 10
        tof_multi: 50
        tof_row: 50
        camera: 100
        cliff: 10
